creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
 
-Click the **Use** button (right-click for Windows 10 OS), to open up the command block user interface.
+Click the **Use** button (right-click on PC) to open up the command block user interface.
 
 ## Command Block User Interface
 
 ![Overview of the command block graphical user interface](Media/CommandBlocks/commandblockgui.png)
 
 - **Block Type** defines what type of command block this is. There are 3 options:
-    - **Impulse** will execute once they are activated via redstone signal.
-    - **Chain** will execute every time that they are triggered by the previous block in the chain.
-    - **Repeat** will execute their command once every in-game tick.
-        - By default, **Minecraft ticks are 20 ticks per second**.
--	**Condition** defines when a command block will execute it’s action. There are 2 options:
-    - **Conditional** will only execute if the previous block in a chain was successful
+    - **Impulse** command blocks are activated when they receive a redstone signal.
+    - **Chain** command blocks are activated when a command block pointing into it is activated.
+    - **Repeat** command blocks are activated once every in-game tick, as long as they have a redstone signal. Minecraft has 20 ticks per second.
+-   **Condition** defines when a command block will execute its action. There are 2 options:
+    - **Conditional** will only execute if the previous block in a chain was successful.
     - **Unconditional** will execute regardless if the previous block was successful or not.
 - **Redstone** defines whether or not a command block requires redstone to activate. There are 2 options:
-    - **Needs Redstone** will set the command block to require redstone activation source.
-    - **Always Active** will set the command block to always activate with or without redstone.
-- **Execute on First Tick** is a boolean (True/False statement) that is used for command blocks set to repeat. If set **True**, the command will activate as soon as it is activated. If set **False**, the command will activate on a delay.
-- **Delay in Ticks** is a value that defines how long a command waits to execute after the command block has been activated.
+    - **Needs Redstone** will set the command block to only activate when it has redstone power.
+    - **Always Active** will set the command block to always activate with or without redstone. For impulse command blocks, this causes them to only run once, so it isn't very useful. For chain command blocks, this is enabled by default.
+- **Execute on First Tick** is only usable for repeating command blocks with a delay. If enabled, the command will execute as soon as the block is powered, and then will wait for the delay before running again. If disabled, the command will not run the first time until the delay has finished.
+- **Delay in Ticks** is a value that defines how long a command block waits to execute after the command block receives redstone power.
```

Comment:
I changed all of these to mostly be more intuitive and clear regarding concepts the average player should already be familiar with.